sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+﻿# Contributing 
+
+Coming Soon...
```

Comment:
Including this since it was linked to in the README.  At present, this is largely a skeleton that will be completed once samples are ready.